libtransmission/session.cc

```diff
@@ -341,6 +341,21 @@ size_t tr_session::WebMediator::clamp(int torrent_id, size_t byte_count) const
     return tor == nullptr ? 0U : tor->bandwidth().clamp(TR_DOWN, byte_count);
 }
 
+std::optional<std::string_view> tr_session::WebMediator::proxyUrl() const
+{
+    return session_->settings_.proxy_url;
+}
+
+void tr_session::WebMediator::notifyBandwidthConsumed(int torrent_id, size_t byte_count)
```

Comment:
Please remove this function, as it was removed in f1102f0b4d34de9ebaa04852c4f9882f549317cc.